test/dataframe.jl

```diff
@@ -965,7 +1002,7 @@ end
     x = collect(1:10)
     y = collect(1.0:10.0)
     z = collect(10:-1:1)
-    df = DataFrame(x = x, y = y)
+    df = DataFrame(:x=>x, :y=>y, copycolumns=false)
```

Comment:
This should probably not be changed. Same below.

Reply:
I will decide when we finalize the decision if `DataFrame` constructor taking only kwargs accepts `copycolumns` kwarg.

Reply:
fixed